pootle/apps/pootle_statistics/migrations/0002_auto_20150903_1841.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please name migration files using a human readable name to describe what its doing, rather then the default timestamp